packages/mattermost-redux/src/store/index.ts

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-const config = require('./configureStore').default;
-export default config;
+import configureStore from './configureStore';
```

Comment:
This simple change has broken a ton of type checking throughout the app because it makes it so that TS can actually check the type of `configureStore` and the store throughout the rest of the code. I might have to revert it for this PR